src/components/featured-article/macro.spec.js

```diff
@@ -0,0 +1,133 @@
+/** @jest-environment jsdom */
+
+import * as cheerio from 'cheerio';
+import axe from '../../tests/helpers/axe';
+import { renderComponent } from '../../tests/helpers/rendering';
+import { EXAMPLE_FEATURED_ARTICLE_WITH_CHART, EXAMPLE_FEATURED_ARTICLE_WITH_IMAGE } from './_test-examples';
+
+describe('Macro: Featured Article', () => {
+    describe('GIVEN: Params: required', () => {
+        describe('WHEN: chart is provided and image is not', () => {
+            const $ = cheerio.load(renderComponent('featured-article', EXAMPLE_FEATURED_ARTICLE_WITH_CHART));
```

Comment:
EXAMPLE_FEATURED_ARTICLE_WITH_CHART and EXAMPLE_FEATURED_ARTICLE_WITH_IMAGE set other non required things such as metadata. For these tests we should only pass the required things and then add in metadata for the metadata test